cav_msgs/msg/TrajectoryPlanPoint.msg

```diff
@@ -0,0 +1,22 @@
+# TrajectoryPlanPoint.msg
+
+# This message describes a 2D point on plugin's trajectory plan
+# It is assmued that this point is in the MAP frame
+
+# X coordinate of the trajectory plan point
+float64 x
+
+# Y coordinate of the trajectory plan point
+float64 y
+
+# Desired time when reaching the trajectory plan point
+uint64 timestamp
+
+# Lane Id of the trajectory plan point
+uint8 lane_id
```

Comment:
I think our lane_ids will be more complex than uint8. We will need some sort of combined road/lane_id or more lanes than just 255 since our routes will be built out of existing HD map lane descriptions.